### ISSUE TITLE
Does the RKI still recommend PCR testing if an exposure with increased risk is shown in the CWA?

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-documentation#720]
## Your Question
* Documentation File: FAQ entry [If I see a high-risk encounter in the App, i.e. a red card, should I do a rapid antigen test or PCR-Test?](https://www.coronawarn.app/en/faq/#rat_red_card_which_test)
* Question: Does the RKI still recommend to get a PCR test if the CWA shows an exposure with increased risk?
Because on https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Warn_App.html they now state this:

<img width="629" alt="Bildschirmfoto 2021-10-14 um 17 40 55" src="https://user-images.githubusercontent.com/67682506/137351253-b6f4e976-3ec7-4d61-b905-de1c7e12d341.png">
(I know, they are talking about the legal situation in the last paragraph.)
Or has the recommendation meanwhile changed to first confer with the doctor and the doctor then decides if a test is necessary and if a test is necessary which test will be done? 

---
Internal Tracking ID: [EXPOSUREAPP-9990](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9990)

**Comment by ndegendogo** [comment]
> Or has the recommendation meanwhile changed to first confer with the doctor and the doctor then decides if a test is necessary and if a test is necessary which test will be done?

@Ein-Tim as far as I remember, consulting a doctor and discuss further proceeding wirh the doctor has **always** been the recommendation in the app.

**Comment by MikeMcC399** [comment]
@ndegendogo 
> as far as I remember, consulting a doctor and discuss further proceeding wirh the doctor has always been the recommendation in the app.

You are right:

![TracingDetailsFragment_tracing_increased_2 11](https://user-images.githubusercontent.com/66998419/137450575-0d0a9544-b829-43ab-9155-b8a520de5b5e.png)

however, and you can't see this on the published screenshot, the app also links to https://www.coronawarn.app/en/faq/#red_card_how_to_test which brings the FAQs into play.

The RKI page [Handreichung des Robert Koch-Instituts für niedergelassene Ärztinnen und Ärzte](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Handreichung-Arzt.html) was last updated more than a year ago on June 24, 2020.

**Comment by MikeMcC399** [comment]
@Ein-Tim 
You have posted the current recommendation from RKI yourself in https://github.com/corona-warn-app/cwa-documentation/issues/720#issue-1026553793, which does not recommend any particular type of test.

The question now is if RKI still wants the [CWA FAQ - #rat_red_card_which_test](https://www.coronawarn.app/en/faq/#rat_red_card_which_test) to say:

"**If I see a high-risk encounter in the App, i.e. a red card, should I do a rapid antigen test or PCR-Test?**

If the Corona-Warn-App shows you an encounter with increased risk (red tile), you should take a PCR test and quarantine yourself.

Please confer with your doctor for the further procedure."

Only RKI can answer that question.

The app recommends contacting:
- Your general practitioner
- The general medical emergency service on telephone number 116 117
- Your public health authority

If you believe that the app is the authority on such statements, then FAQ articles like [#rat_red_card_which_test](https://www.coronawarn.app/en/faq/#rat_red_card_which_test) should use the same language and the same list of recommended contacts.

The community can draw attention to discrepancies like this, but the responsibility lies with RKI to define what messages should be given to users.

**Comment by Jo-Achim** [comment]
Of course, I cannot answer for the RKI, but I can refer to the current test regulation: 
[Verordnung zum Anspruch auf Testung in Bezug auf einen direkten Erregernachweis des Coronavirus SARS-CoV-2 (Coronavirus-Testverordnung - TestV) ,§ 2 Testungen von Kontaktpersonen](https://www.gesetze-im-internet.de/coronatestv_2021-10/__2.html).
> 5. People who have received a warning from the Robert Koch Institute's Corona warning app with the status display of increased risk,
(Personen, die durch die Corona-Warn-App des Robert Koch-Institutes eine Warnung mit der Statusanzeige erhöhtes Risiko erhalten haben,)

From my own experience I can say that this applies to rapid tests and PCR tests. 

Cross reference: https://github.com/corona-warn-app/cwa-wishlist/issues/676#issue-1048331229.

**Comment by MikeMcC399** [comment]
The FAQ 
- https://www.coronawarn.app/de/faq/#rat_red_card_which_test is still out of sync with the RKI site
- https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Warn_App.html "Statusanzeigen in der Corona-Warn-App". 

**The two sites should not be giving different recommendations, especially also considering they are both owned by RKI.**

- https://www.coronawarn.app/de/imprint/
- https://www.rki.de/DE/Service/Impressum/impressum_node.html

Below in German for 1:1 comparison.

---
https://www.coronawarn.app/de/faq/#rat_red_card_which_test "Wenn ich eine hohe Risikobegegnung in der App angezeigt bekomme, d. h. eine rote Karte/Kachel, sollte ich dann einen Schnelltest oder PCR-Test machen?"

![rat_red_card_which_test_DE](https://user-images.githubusercontent.com/66998419/142610996-43f12905-2b0b-49ee-afb6-0a2fa911bdd3.jpg)

---

https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Warn_App.html "Statusanzeigen in der Corona-Warn-App"

![RKI CWA erhöhtes Risiko](https://user-images.githubusercontent.com/66998419/142611338-881db1a6-9166-4ef9-881a-ee8456d06cd6.jpg)

**Comment by Ein-Tim** [comment]
The (simple) question stays: Does the RKI still recommend to get tested via PCR test if CWA shows an increased risk warning.

**Comment by Ein-Tim** [comment]
@dsarkar As this issue is slightly related to https://github.com/corona-warn-app/cwa-website/issues/2607 & https://github.com/corona-warn-app/cwa-documentation/issues/877, could you please check if we can also make changes here? Thanks!

**Comment by dsarkar** [comment]
@Ein-Tim Will check.

**Comment by svengabr** [comment]
I have just checked the linked Jira issue.

Jira Ticket is flagged as:
Resolution: `Fixed`
Status: `Completed`

**Comment by Ein-Tim** [comment]
@svengabr 

But my question was never answered 🥲

**Comment by svengabr** [comment]
@Ein-Tim I have two pieces of information available:

1. A mail from RKI regarding this ticket (in german):
![rki](https://user-images.githubusercontent.com/4686874/173826126-6805a697-8fdd-483a-b143-8d1ae34c2ad8.png)

2. A PR that you have created yourself https://github.com/corona-warn-app/cwa-website/pull/1972

Does the mail from RKI answer your question? If not, what do you think is still open from your side?

**Comment by Ein-Tim** [comment]
@svengabr 

My question is about https://www.coronawarn.app/en/faq/#rat_red_card_which_test, where the recommendation is to take a PCR test.

**Comment by MikeMcC399** [comment]
- See also https://github.com/corona-warn-app/cwa-website/issues/2962.

If the regulations do not change, then https://www.coronawarn.app/en/faq/#rat_red_card_which_test will become incorrect on July 1, 2022 due to expiry of [TestV](https://www.gesetze-im-internet.de/coronatestv_2021-10/) "Verordnung zum Anspruch auf Testung in Bezug auf einen direkten Erregernachweis des Coronavirus SARS-CoV-2 (Coronavirus-Testverordnung - TestV)" according to [§ 19 Inkrafttreten, Außerkrafttreten](https://www.gesetze-im-internet.de/coronatestv_2021-10/__19.html) "(1) Diese Verordnung tritt am 11. Oktober 2021 in Kraft; sie tritt am 30. Juni 2022 außer Kraft."

**Comment by Ein-Tim** [comment]
Please reopen this issue. The question I asked was never answered.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

What about the RKI answer in https://github.com/corona-warn-app/cwa-documentation/issues/877#issuecomment-1155048115 from @larswmh ?

>Developer comment:
>
>Issue was discussed in today's RKI call. Feedback: While the finding is in general >correct, RKI's interpretation is still that people with a red risk card should get a >free antigen test. As there is anyhow some changes in regards to free of charge >antigen tests expected in the next months, we dont't do any changes now.

RKI's page https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Teststrategie/Nat-Teststrat.html links to https://www.bundesgesundheitsministerium.de/coronavirus/angepasste-teststrategie.html
where it says:

"Eine rote Warnmeldung auf der Corona-Warn-App reicht nicht mehr aus, um einen PCR-Test zu bekommen. Ein Anspruch muss zunächst mit einem Antigentest abgeklärt werden."

There is also a YouTube video https://youtu.be/iGFuLoTR4Ac (which is now a bit old) explaining about it.

This is only valid for the next two weeks until the regulations expire.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I nowhere find the confirmation that the RKI is actually still recommending PCR tests.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

> I nowhere find the confirmation that the RKI is actually still recommending PCR tests.

There is no confirmation of that statement. It says they recommend a Rapid Antigen Test followed by a PCR test if the Rapid Antigen Test is positive.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Right, so should we change the FAQ?

**Comment by MikeMcC399** [comment]
@Ein-Tim 

> so should we change the FAQ?

- There are inconsistencies in the FAQs, so I opened a new issue so these can be reviewed. See https://github.com/corona-warn-app/cwa-website/issues/2965

**Comment by ndegendogo** [comment]
Maybe the wording should be different.

- if you have any symptoms, there is already a strong recommendation to consult a doctor for further advice
- if the doctor then recommends a PCR, the cost will be covered by your health insurance (after all, you are sick)
- if you have a red warning, but no symptoms, you can (currently) get a RAT for free; but no PCR.
- if you have a positive RAT, you can (currently) get a PCR for free - *independent of symptoms, and independent of cwa red warnings*
- test centers that offer PCR for self-payers are rare (at least in my region)
- in my region I can (currently) get a RAT for free daily - *independent of any symptoms, and independent of any red warning*. **This is for sure different in other regions.**

So, to sum it up, red warnings don't make any difference for me meanwhile....

**Comment by Ein-Tim** [comment]
@ndegendogo 

I agree with you! Still, an updated recommendation or a confirmation from the RKI shouldn't be that hard to get, or...

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> Still, an updated recommendation or a confirmation from the RKI shouldn't be that hard to get

- Since this is now seems to be about the wording in the FAQs, I suggest to continue the conversation in https://github.com/corona-warn-app/cwa-website/issues/2965. I like the idea to include more detail like @ndegendogo suggested, also including a different recommendation if somebody has symptoms or has no symptoms.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

This was always about the FAQ as you can see in the OP.

I'd like to wait until we hear feedback from @svengabr or his team.

I also like @ndegendogo's idea!
